src/cpp/common/channel_arguments.cc

```diff
@@ -81,6 +90,28 @@ void ChannelArguments::SetCompressionAlgorithm(
   SetInt(GRPC_COMPRESSION_ALGORITHM_ARG, algorithm);
 }
 
+// Note: a second call to this will add in front the result of the first call.
```

Comment:
It would help to add more details/context to this comment. Until you pointed me to the issues (https://github.com/grpc/grpc/issues/4069 and https://github.com/grpc/grpc/issues/2661), i thought calling it twice is a "bug" and should be avoided :).

Perhaps adding a bit more context will help - or just saying that user should call it twice if they want to add more user agent identifiers..
 
>>> (pasting this from #2661)
"RFC 7231§5.5.3 says:
By convention, the product identifiers are listed in decreasing order of their significance for identifying the user agent software.
Combined with the example, it seems that we should put the application-provided string first."
<<<